### ISSUE TITLE
`ocdev application list` does not show application, but `get` does`

### ISSUE THREAD

**Issue by containscafeine** [redhat-developer/ocdev#142]
```console
$ ocdev application list 
ACTIVE   NAME

$ ocdev application get
The current application is: app
```

**Comment by kadel** [comment]
Yeh, this is kind of tricky :-(

We need to define what is the right behavior if there is no application.
So the list is listing existing applications, and it is empty because there is no app.

'get' is showing 'app' because by definition if there is no application the default application should be used, and default application is called 'app'

**Comment by containscafeine** [comment]
@kadel this is why we should separate the setting default logic from GetCurrent() :)

**Comment by kadel** [comment]
> @kadel this is why we should separate the setting default logic from GetCurrent() :)

That is not about that. It is easy to do it. Question is what is the right way to present it!

**Comment by kadel** [comment]
closed in #215
